### PR TITLE
Support for exit nodes

### LINKED ISSUE #60: Support of using exit nodes
There's currently no way to find and use exit nodes which
makes NOnion unusable for connecting to non-hs servers.

### PR DESCRIPTION
**Pull request by aarani**
Fixes #60

### CODE REVIEW

**Comment by aarani** [comment]
> Use Fixes, not Closes.
> 
> Closes is for when the commit doesn't fix the issue, but there's nothing else to do for the issue (e.g. just adding a regression test because the issue was already fixed).

Fixed.

File: NOnion/Directory/TorDirectory.fs [hunk 1]
```diff
@@ -174,6 +175,9 @@ type TorDirectory =
                        Seq.filter(fun router ->
                            Seq.contains "Guard" router.Flags
                        )
+                   | Exit ->
+                       Seq.filter(fun router -> Seq.contains "Exit" router.Flags
```
Comment (knocte) [inline NOnion/Directory/TorDirectory.fs:179]: @aarani yeeesh, this is such a shitty formatting, can you file a bug in fantomless please

Comment (aarani) [reply NOnion/Directory/TorDirectory.fs:179]: Will do it as soon as I was able to cook up a minimal test case

Comment (knocte) [reply NOnion/Directory/TorDirectory.fs:179]: Ok, do you mind I don't merge until that happens, so that we don't forget about the issue. Thanks

Comment (aarani) [reply NOnion/Directory/TorDirectory.fs:179]: https://github.com/nblockchain/fantomless/issues/9

**Review by knocte — APPROVED** [review]

**Pull request merged**